### ISSUE TITLE
Prevent forced use of <fieldset> for individual fields.

### ISSUE THREAD

**Issue by willnationsdev** [techniq/svelte-ux#108]
I noticed while using the package that TextField and SelectField both have a `<fieldset>` at the root while most other inputs have a `<div>`. This causes problems for me in a codebase I am attempting to use this package for since I need to be able to put multiple inputs/fields into one fieldset (and also include a `<legend>` as a header for that subset of fields).

Would it be possible to update those two (and any others I missed) to likewise swap `<fieldset>` for `<div>` so that users can handle that aspect themselves? Or if you don't want to break compatibility, perhaps use `<svelte:element this={rootElement}>` with an `export let rootElement = "fieldset"`?

I'd be happy to submit a PR myself if approved.

**Comment by techniq** [comment]
@willnationsdev I don't have any strong reasons for keeping `fieldset` as the root instead of a standard `div`.  Originally I thought it made more semantic sense, and was easier to target via CSS (and spot in dev tools), but I think it was an incorrect assumption.  Since each component now has the component name as the class, most of those reasons are negated.

While a [Field](https://github.com/techniq/svelte-ux/blob/master/src/lib/components/Field.svelte#L58) can have more than 1 control, in practice there isn't a guarantee.  I wonder if it too should just become a div?

One of the main reason's `TextField` doesn't use `Field` internally is because of https://github.com/sveltejs/svelte/issues/6059

So in short, I'm all for changing it to a `div` (and maybe instances) especially if it's causing you issues. 

A PR would be awesome, and maybe someone else will chime in before we review/merge.  Library is pre-1.0 for these types of revisions 😃

**Comment by techniq** [comment]
Heads up, `Field` also exposes a `fieldset` [slot](https://github.com/techniq/svelte-ux/blob/master/src/lib/components/Field.svelte#L201), which is used by [MenuField](https://github.com/techniq/svelte-ux/blob/master/src/lib/components/MenuField.svelte#L105).  Would need to come up with a better name for this :)

**Comment by willnationsdev** [comment]
> Heads up, `Field` also exposes a `fieldset` [slot](https://github.com/techniq/svelte-ux/blob/master/src/lib/components/Field.svelte#L201), which is used by [MenuField](https://github.com/techniq/svelte-ux/blob/master/src/lib/components/MenuField.svelte#L105). Would need to come up with a better name for this :)

Hmmm. Perhaps "root", "container", or "shell"?

**Comment by techniq** [comment]
I like `root`, which aligns with `classes.root` and is used through Svelte UX to style the top-most element (and `classes.container` can be used within a component like Field does :)).  Thanks
